### PR TITLE
refactor: expose ref

### PR DESCRIPTION
**Pull request by jefrydco**
## Issue

https://github.com/jitera/jitera-frontend/issues/1248

### CODE REVIEW

File: src/components/atoms/Button/Component.tsx [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
   title?: string;
   titleStyle?: StyleProp<TextStyle>;
-  disabled?: boolean;
   loading?: boolean;
-  onPress?: (event: GestureResponderEvent) => void;
-}>;
+}
 
-const Button: FC<ButtonProps> = forwardRef<any, ButtonProps>(
+const Button = React.forwardRef<TouchableOpacity, ButtonProps>(
```
Comment (quy-truong) [inline src/components/atoms/Button/Component.tsx:20]: nice 👍

File: src/components/atoms/Divider/Component.tsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
   size?: number;
   content?: string | (() => JSX.Element);
   contentPosition?: 'left' | 'center' | 'right';
   contentStyles?: StyleProp<ViewStyle>;
   containerStyles?: StyleProp<ViewStyle>;
-}>;
+}
 
-const Divider: FunctionComponent<DividerProps> = forwardRef<any, DividerProps>(
+const Divider = React.forwardRef<any, DividerProps>(
```
Comment (quy-truong) [inline src/components/atoms/Divider/Component.tsx:17]: can we fix `any` here? 🤔

File: src/components/atoms/Image/Component.web.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ export type ImageProps = PropsWithRef<
   }
 >;
 
-const Image: FC<ImageProps> = forwardRef<any, ImageProps>(
+const Image = React.forwardRef<any, ImageProps>(
```
Comment (quy-truong) [inline src/components/atoms/Image/Component.web.tsx:12]: can we fix `any` here?

File: src/components/atoms/Page/Component.tsx [hunk 4]
```diff
@@ -10,68 +10,72 @@ import {
...
+   * Enable to draw behind status bar android
+   * @default false
+   */
+  draw?: boolean;
 
-    /**
-     * Enable to custom background color of inset bottom on Iphone
-     * @default false
-     */
-    insetBottom?: boolean;
```
Comment (quy-truong) [inline src/components/atoms/Page/Component.tsx:43]: do we need to update in our `jitera-frontend`? 🤔

**Review by quy-truong — APPROVED** [review]

File: src/components/atoms/Page/Component.tsx (hunk 4 above)
Comment (jefrydco) [reply src/components/atoms/Page/Component.tsx:43]: Yes, I'll update as well

File: src/components/atoms/Page/Component.tsx [hunk 5]
```diff
@@ -10,68 +10,72 @@ import {
...
-    unsafe?: boolean;
-    hidden?: boolean;
-    statusColor?: string;
-    safeArea?: boolean;
-    /**
-     * Enable to draw behind status bar android
-     * @default false
-     */
-    draw?: boolean;
+import type { SafeAreaView } from 'react-native-safe-area-context'
```
Comment (zrg-team) [inline src/components/atoms/Page/Component.tsx:17]: @jefrydco don't used it directly here

Comment (zrg-team) [reply src/components/atoms/Page/Component.tsx:17]: and SafeAreaView: typeof SafeAreaView;
canbe View or any component note typeof SafeAreaView

**Review by zrg-team — CHANGES_REQUESTED** [review]

**Review by duongjitera — APPROVED** [review]

**Review by zrg-team — APPROVED** [review]

**Pull request merged**